pytest_tests/testsuites/failovers/test_failover_network.py

```diff
@@ -110,3 +114,56 @@ def test_block_storage_node_traffic(
                 wallet, cid, oid, shell=self.shell, endpoint=new_nodes[0].get_rpc_endpoint()
             )
             assert get_file_hash(source_file_path) == get_file_hash(got_file_path)
+
+    @pytest.mark.sanity
+    @allure.title("RPC reconnection test")
+    def test_rpc_reconnection(self, hosting: Hosting):
+        """
+        When RPC connection fails (and it can), storage node reconnects to some other node and continues to operate.
+        """
+        dport_repeat = 50  # Constant for the number of the disconnect should be repeated
+        morph_chain_endpoints = get_morph_chain_endpoints(hosting)
+
+        for storage_node in hosting.find_service_configs(STORAGE_NODE_SERVICE_NAME_REGEX):
+            host = hosting.get_host_by_service(storage_node.name)
+            pid = host.get_service_pid(storage_node.name)
+
+            for morph_chain_addr, morph_chain_port in morph_chain_endpoints:
+                with allure.step(f'Disconnecting storage node {storage_node.name} '
+                                 f'from {morph_chain_addr} {dport_repeat} times'):
+                    for _ in range(dport_repeat):
```

Comment:
You don't have any delay between disconnection attempts here, so it's very likely that the loop will perform real disconnection fewer number of times.

Reply:
> You don't have any delay between disconnection attempts here, so it's very likely that the loop will perform real disconnection fewer number of times.

@roman-khimov What delay do we expect in reality?
A few seconds?

Reply:
Likely to be in ms range, so a second should be enough.

Reply:
> Likely to be in ms range, so a second should be enough.

Done